modules/react/popup/lib/PopupCloseIcon.tsx

```diff
@@ -21,7 +21,7 @@ export const PopupCloseIcon = createSubcomponent('button')({
       icon={xIcon}
       type="button"
       position="absolute"
-      right={space.xxxs}
+      insetInlineEnd={space.xxxs}
```

Comment:
nice!

Reply:
This is what we'll be moving to with other components as well so we no longer have to worry about RTL.